app/(tabs)/index.tsx

```
@@ -86,6 +86,10 @@ const HomeScreen = () => {
     calculateBMI();
   }, [initializeHealthTracking]);
 
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on bmi.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)